src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -52,6 +57,37 @@ export const WorkspaceList = () => {
     [application, http]
   );
 
+  const handleUpdateWorkspace = useCallback(
+    (id: string) => {
+      if (application && http) {
+        updateWorkspace({ application, http }, id);
+      }
+    },
+    [application, http]
+  );
+
+  const searchWorkspaces = useCallback((query: string, list: WorkspaceAttribute[]) => {
```

Comment:
Seems this searchWorkspaces only be used in below useMemo, how about move this function inside useMemo?